services/pages_api/src/app.py

```diff
@@ -0,0 +1,45 @@
+"""
+Page API endpoints
+"""
+import pymongo
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+from flask import Flask, request, abort
+from flask.json import JSONEncoder
+import os
+import logging
+logging.basicConfig(format='%(levelname)s :: %(asctime)s :: %(message)s', level=logging.DEBUG)
+from bson import json_util
+import base64
+import json
+from flask import jsonify
+
+app = Flask(__name__)
+
+PAGE_SIZE = 200
```

Comment:
Can you make this an argument in the request?

Reply:
Yes